### ISSUE TITLE
Interaction claims are linked to an interaction claim view, which is broken

### ISSUE THREAD

**Issue by susannasiebert** [griffithlab/dgi-db#235]
E.g. on this page: http://localhost:3000/interactions/c1d6f2d8-56ff-4420-b909-6c902ab19b2f#_claims. The box for the interaction claim is a link to the interaction claim view, which is currently broken. This link should be removed and we shouldn't have a view for an individual interaction claim.

**Comment by yanyangfeng** [comment]
This is fine now, correct?

**Comment by ahwagner** [comment]
Yes, we're currently not linking to the interaction claim view. However, we will, at some point, wish to fix this and restore these links. With the removal of links to the interaction claims, this is no longer a high-priority issue.
